src/components/pages/Team.jsx

```diff
@@ -1,28 +1,94 @@
 // @flow
 
-import * as React from 'react';
-
+import React, {useState} from 'react';
+import {withLocale} from '../../utils/locale/withLocale';
+import Modal from '../modal/Modal';
 import style from './page.css';
 
-const Team = () => (
-  <>
-    <div className={style.hero} />
-    <div className={style.content}>
-      <h1 className={style.headline}>Team</h1>
-
-      <div className={style.teamFlexContainer}>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
-        <div className={style.teamItem}>item</div>
+type TeamPropsType = {
+  translate: string => string,
+};
+
+const Team = ({translate}: TeamPropsType) => {
+  const [modalVisible, showModal] = useState(false);
+  const [itemShown, setActiveItem] = useState(null);
+
+  const openModal = id => {
+    modalVisible ? showModal(false) : showModal(true);
+    itemShown ? setActiveItem(null) : setActiveItem(id);
+    console.log('open modal', id, itemShown);
+  };
+
+  const array = [
```

Comment:
this can be further shortened:
```
const images = ['prada', 'emaj', 'krzysiek', ....];


<img src={`../../images/team/${name}_thumb.jpg`} />
<Modal
  id={itemShown}
  modalVisible={modalVisible}
  name={translate(`TEAM_${index}_NAME`)}
  info={translate(`TEAM_${index}_CONTENT`)}
  imagePath={`../../images/team/${name}_large.jpg`}
/>
```

Up to you if you wish to update it 👍